src/libs/actions/Wallet.js

```diff
@@ -153,6 +154,70 @@ function buildIdologyError(idologyErrors) {
     return `${errorStart} ${Localize.arrayToString(errorsTranslated)}. ${errorEnd}`;
 }
 
+/**
+ * Validates a user's provided details against a series of checks
+ *
+ * @param {Object} [personalDetails]
```

Comment:
The object is not optional. You've not given it a default, hence you can update this to,
```
@param {Object} personalDetails
```